### PR TITLE
add webcams and WineAwards config files

### PR DESCRIPTION
**Pull request by BlueRyse**

### CODE REVIEW

File: databrowser/src/config/tourism/webcamInfo/webcamInfo.listView.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import { ListViewConfig } from '../../../domain/datasetConfig/types';
+import { CellComponent } from '../../../domain/cellComponents/types';
+
+export const eventTopicsListView: ListViewConfig = {
```
Comment (gappc) [inline databrowser/src/config/tourism/webcamInfo/webcamInfo.listView.ts:4]: Variable name seems wrong - should probably be something like "webcamInfoListView"

File: databrowser/src/config/tourism/webcamInfo/webcamInfo.views.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { eventTopicsListView } from './webcamInfo.listView';
+
+export const webcamInfoViews = {
+  table: eventTopicsListView,
```
Comment (gappc) [inline databrowser/src/config/tourism/webcamInfo/webcamInfo.views.ts:4]: Please check the variable name (see other comment)

**Review by gappc — CHANGES_REQUESTED** [review]

**Comment by BlueRyse** [comment]
@gappc Lukas fixed the naming issues, can this be merged now?

**Comment by BlueRyse** [comment]
@gappc Nevermind, I saw the new Webcams listview configuration. Let me implement that and then we can merge.

**Comment by gappc** [comment]
Hi @BlueRyse, the merge is done, thx for your work.

There is no dedicated Cell Component for URLs at the moment, we could implement one if necessary. I've started a new discussion (see #120), it would be nice if you could add yout use case.

**Pull request merged**